src/bootstrap/utils.ts

```diff
@@ -138,21 +139,35 @@ async function bootstrapState({
     const revProxy = new ReverseProxy({
       logger: logger.getChild(ReverseProxy.name),
     });
-    const nodeManager = await NodeManager.createNodeManager({
+    const nodeGraph = await NodeGraph.createNodeGraph({
       db,
+      fresh,
       keyManager,
-      sigchain,
+      logger: logger.getChild(NodeGraph.name),
+    });
+    const nodeConnectionManager = new NodeConnectionManager({
+      keyManager,
+      nodeGraph,
+      // SeedNodes: undefined, // I don't think this is needed for bootstrapping.
       fwdProxy,
       revProxy,
+      logger: logger.getChild(NodeConnectionManager.name),
+    });
+    await nodeConnectionManager.start();
```

Comment:
Does this need to be started? If the side-effects of starting the `NodeConnectionManager` isn't persistent, it doesn't actually need to be started. Remember the bootstrap process just shutsdown immediately after setting all the on-disk side-effects.

Reply:
I suppose it doesn't have to be started there.

Reply:
It would add the seedNodes to the `NodeGraph` but I don't think that's needed for bootstrapping. I've removed it.